GWAS/PCA.ipynb

```diff
@@ -928,10 +928,10 @@
     "parameter: k = 10\n",
     "# Number of Principal Components based on which outliers should be evaluated. Default is 5 but this should be based on examine the scree plot\n",
     "parameter: maha_k = 5\n",
-    "# Homogeneity of populations \n",
+    "# Homogeneity of populations. Set to True if your pops are homogeneous\n",
```

Comment:
Just to clarify, here you realize the correct command argument should be `--homogeneous` when you need it `True` and `--no-homogeneous` when you need it `False`, right?

Reply:
I wasn't sure about this